lib/generators/provider/USAGE

```diff
@@ -0,0 +1,5 @@
+Description:
+    Create or update provider
+
+Example:
+    rails generate provider VendorName
```

Comment:
Looks good, it would be great to add more info here if possible.  From a high level, I'm not sure what you have to do before and after you generate a provider.  Which of the many generated files should they look at, etc.